packages/eas-cli/src/project/publish.ts

```diff
@@ -285,12 +286,18 @@ export async function uploadAssetsAsync(
   const totalAssets = uniqueAssets.length;
 
   updateSpinnerText?.(totalAssets, totalAssets);
-
   let missingAssets = await filterOutAssetsThatAlreadyExistAsync(uniqueAssets);
   const uniqueUploadedAssetCount = missingAssets.length;
-  const { specifications } = await PublishMutation.getUploadURLsAsync(
-    missingAssets.map(ma => ma.contentType)
-  );
+
+  const missingAssetChunks = chunk(missingAssets, 100);
```

Comment:
How well do we handle 1000?

Reply:
We handle 1400 in prod relatively ok (haven't seen failures), 100 is more just setting this to a value that doesn't make this too slow for normal apps (1-2 requests for most apps I assume) while keeping us far away from hitting the request timeout.

Reply:
OK. My thinking is that we want to find a sweet spot that uses batching efficiently while also not overloading a replica and causing it to run for too long (~1 second would be a high target IMO).